io/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: CC0-1.0
+
```

Comment:
Typo on line 7 "minmal" should be "minimal"

Reply:
Thanks man, I swear I fixed that already when hacking on this PR.